bin/adhoc-scripts/checkStuckLQE.py

```diff
@@ -72,9 +72,9 @@ def commonDataLocation(element):
     return commonLoc
 
 def main():
-    if 'WMAGENT_CONFIG' not in os.environ:
-        os.environ['WMAGENT_CONFIG'] = '/data/srv/wmagent/current/config/wmagent/config.py'
-    config = loadConfigurationFile(os.environ["WMAGENT_CONFIG"])
+    if 'WMA_CONFIG_FILE' not in os.environ:
+        os.environ['WMA_CONFIG_FILE'] = '/data/srv/wmagent/current/config/config.py'
+    config = loadConfigurationFile(os.environ["WMA_CONFIG_FILE"])
```

Comment:
Just a different way to accomplish the same thing, but with less lines of code :) How about remove the 2 lines above and using the following construction (with a default value, in case the env var is not set):
```
os.environ.get('WMA_CONFIG_FILE', '/data/srv/wmagent/current/config/config.py')
```

Reply:
Sounds great to me, I will use it